### ISSUE TITLE
Fast unloading produces a warning "Warning: Can't call setState (or forceUpdate) on an unmounted component"

### ISSUE THREAD

**Issue by clytras** [archriss/react-native-render-html#183]
### Is this a bug report or a feature request?

Bug report

### Have you read [the guidelines](https://github.com/archriss/react-native-render-html/blob/master/CONTRIBUTING.md) regarding bug report?

Yes

### Have you read [the documentation](https://github.com/archriss/react-native-render-html/blob/master/README.md) in its entirety?

Yes

### Have you made sure that your issue hasn't already been reported/solved?

Yes

### Is the bug specific to iOS or Android? Or can it be reproduced on both platforms?

iOS/Android all versions

### Is the bug reproductible in a production environment (not a debug one)?

Yes

### Have you been able to reproduce the bug in [the provided example](https://github.com/archriss/react-native-render-html/tree/master/Demo)?

No. It does not have navigation changing screens.

### Environment

  Environment:
    React: 16.4.1
    React native: 0.56.0
    react-native-render-html: 3.10.0

  Target Platform:
    Android (8.1)
    iOS (11.4)

### Steps to Reproduce

1. Create a nanigation based RN app
2. Create a new navigation view with `<HTML ... />` component rendering
3. First navigate to that screen let it load and go back to the first screen, nothing happens
4. Then navigate to that same screen aggain, but now don't let it finish loaging and hit back quickly, it displays a notice.

### Expected Behavior

Not expecting any RN warnings to popup while quickly navigating back from a view containing `<HTML .../>` component before it finishes loading.

### Actual Behavior

A warning appears in the console debug window:

```
Warning: Can't call setState (or forceUpdate) on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in HTMLImage (at HTMLRenderers.js:48)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in HTML (at NewsView.js:37)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in RCTScrollView (at ScrollView.js:1070)
    in ScrollView (at NewsView.js:55)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in ViewBase (at NewsView.js:49)
    in NewsView (at gestureHandlerRootHOC.android.js:29)
    in GestureHandlerRootView (at gestureHandlerRootHOC.android.js:28)
    in Wrapper (at ComponentWrapper.js:27)
    in WrappedComponent (at renderApplication.js:32)
    in RCTView (at View.js:43)
    in RCTView (at View.js:43)
    in AppContainer (at renderApplication.js:31)
```

### Reproducible Demo

No demo, but here is a screen gif capture:

<img src="https://zikro.gr/img/react-native-render-html-bug.gif" alt="react-native-render-html-bug" width="100%"/>

The first navigation does not produce any warnings, but the second one, which navigates back before the component finish loading, produces a warning at `HTMLImage`. Maybe it's because it tries to fetch the image size with a promise and the component is already unmounted when the image loading is done.

**Comment by luutruong** [comment]
I have same this issue as well 🗡

**Comment by Exilz** [comment]
Fixed in `4.1.0`
